tests/conftest.py

```diff
@@ -25,7 +25,10 @@ def users(app, db):
     """Create admin, librarians and patrons."""
     # with Postgresql, when dropping the User table, the sequence is not
     # automatically reset to 1, causing issues with the tests demo data.
-    db.session.execute("ALTER SEQUENCE IF EXISTS accounts_user_id_seq RESTART")
+    if db.engine.driver != "pysqlite":
```

Comment:
I had to add this, the tests were failing on the incorrect syntax for sqlite

Reply:
Yes, but tests should not run on `sqllite`... why do you need it?

Reply:
I thought they are running, then something is wrong with the tests configuration, because it runs sqlite for me and it raises the exeption